### PR TITLE
@ckeditor/ckeditor__*: Fix imports of v27.1

### PR DESCRIPTION
**Pull request by fedemp**
https://github.com/ckeditor/ckeditor5/tree/6e44d34449a95c93c01170a079bdd3c75c3eab5f/packages

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ckeditor/ckeditor5/tree/6e44d34449a95c93c01170a079bdd3c75c3eab5f/packages
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ckeditor__ckeditor5-adapter-ckfinder/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as UploadAdapter } from "./src/uploadadapter";
+import UploadAdapter from './src/uploadadapter';
+
+declare const _default: {
+    UploadAdapter: typeof UploadAdapter;
+};
+
+export default _default;
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-adapter-ckfinder/index.d.ts:12]: I think this should be just:
```suggestion
export {
	UploadAdapter
}
```
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-adapter-ckfinder/src/index.js#L10

File: types/ckeditor__ckeditor5-alignment/index.d.ts [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as Alignment } from "./src/alignment";
-export { default as AlignmentEditing } from "./src/alignmentediting";
-export { default as AlignmentUI } from "./src/alignmentui";
+import Alignment from './src/alignment';
+import AlignmentEditing from './src/alignmentediting';
+import AlignmentUI from './src/alignmentui';
+
+declare const _default: {
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-alignment/index.d.ts:10]:
```ts
export {
	Alignment,
	AlignmentEditing,
	AlignmentUI,
}
```
there is no defualt export IMO:
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-alignment/src/index.js

File: types/ckeditor__ckeditor5-autoformat/index.d.ts [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as Autoformat } from "./src/autoformat";
+import Autoformat from './src/autoformat';
+
+declare const _default: {
+    Autoformat: typeof Autoformat;
+};
+
+export default _default;
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-autoformat/index.d.ts:12]:
```suggestion

export { Autoformat };
```
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-autoformat/src/index.js#L10

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@fedemp thx!

Sorry for noice, I was not expanding lines with an usage, that shows default exports usages

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**